src/Select.js

```diff
@@ -599,4 +654,15 @@ function selectClefs(el, dragHandler){
     }
 }
 
+/**
+ * Check if neume component is part of a ligature
+ * @param {*} nc - The neume component to check.
```

Comment:
Specify the type. If it's the jquery object (it seems to be that), then at least say `{object}`.